### ISSUE TITLE
Schema error ("kilobytes_requested_time" does not exist) during installation of sGMS make target 'done-host/db-schema'

### ISSUE THREAD

**Issue by malachig** [genome/gms#61]
Seeing the following error during installation:

psql:setup/schema.psql:1473: ERROR: column "kilobytes_requested_time" does not exist

**Comment by gatoravi** [comment]
An index is being created for the table 'disk.allocation' based on the column 'kilobytes_requested_time'. The most recent table definition inside the TGI does not have this column for disk.allocation. The lines creating this index need to be removed from the database dump i.e schema.psql

```
-Name: kilobytes_requested_time_idx; Type: INDEX; Schema: disk; Owner: genome; Tablespace: 
-CREATE INDEX kilobytes_requested_time_idx ON allocation USING btree (kilobytes_requested_time);
```

This is fixed in 29f1bfb and has to be tested with a new install of sGMS.

The way to dump the table schemas within the tgi is as follows

```
%ssh gms-postgres
%pg_dump -s genome --username gms-user --host gms-postgres -N workflow -N test_dependencies -T appmanager
```

From Tony,
Use the -N option for pg_dump to skip the workflow schema.  Access to that schema is through a different user, and we're not using the workflow schema on postgres anyway.

Use -N for test_dependencies as well.

**Comment by gatoravi** [comment]
I don't see this error in a fresh install. Closing for now.
